src/utils/shared.ts

```diff
@@ -156,7 +156,7 @@ export const clearEmptyObjectValues = (obj: any): any => {
  * @returns
  */
 export const maskValue = (value: string) => {
-  return '*'.repeat(value.length);
+  return '*'.repeat(value.length).substring(0, 5);
```

Comment:
@uidoyen I guess the same idea applies here, doesn't it?  Also from a quick peek locally this function also is used just once in StepReview.tsx, maybe we don't even need it if you apply the same approach as in the other file.

Reply:
@gashcrumb I thought so but since it was in multiple places in StepReview.tsx I thought to leave it as it was. Let me make it a local function.

Reply:
Cool, that sounds good, thanks!